org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/support/NeighborhoodSignalMessage.java

```diff
@@ -1,11 +1,27 @@
 package org.code.neighborhood.support;
 
 import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
 import org.code.protocol.ClientMessage;
 import org.code.protocol.ClientMessageType;
 
 public class NeighborhoodSignalMessage extends ClientMessage {
   public NeighborhoodSignalMessage(NeighborhoodSignalKey key, HashMap<String, String> detail) {
     super(ClientMessageType.NEIGHBORHOOD, key.toString(), detail);
   }
+
+  @Override
+  public boolean shouldAlwaysSend() {
+    String signalKey = this.getValue();
+    Set<String> ignoredSignalKeys = new HashSet<>();
+    // These keys are only used for testing, don't send in run mode.
```

Comment:
Is this method only called when in "run mode"?

Reply:
I switched the naming convention partway through, I'll clean this up

Reply:
updated the comment, let me know if it makes more sense